src/Mutator/Util/MutatorProfile.php

```diff
@@ -159,6 +160,11 @@
         Mutator\Cast\CastString::class,
     ];
 
+    public const UNWRAP = [
+        Mutator\Unwrap\UnwrapArrayMap::class,
+        Mutator\Unwrap\UnwrapArrayFilter::class,
```

Comment:
@akondas @borNfreee @sanmai 

Is the order important here?

If not, maybe we can sort alphabetically?

Reply:
Same order [as in the manual](http://php.net/manual/en/book.array.php) should work best, IMO.